### PR TITLE
Add validation for username and rpm/tpm negative value

### PR DESCRIPTION
**Pull request by varungup90**
#234

### CODE REVIEW

File: pkg/utils/users.go [hunk 1]
```diff
@@ -54,6 +54,10 @@ func GetUser(u User, redisClient *redis.Client) (User, error) {
 }
 
 func SetUser(u User, redisClient *redis.Client) error {
+	if u.Rpm < 0 || u.Tpm < 0 {
+		return fmt.Errorf("rpm or tpm can not negative")
```
Comment (Jeffwan) [inline pkg/utils/users.go:58]: what's the http error to the client for this case now?

Comment (varungup90) [reply pkg/utils/users.go:58]: internal server error: 500

Comment (Jeffwan) [reply pkg/utils/users.go:58]: should it be something like invalid argument? 400

Comment (varungup90) [reply pkg/utils/users.go:58]: Good point. I will audit all endpoints and their return error codes.

**Pull request merged**